Kickstarter-iOS/Views/Cells/PledgeShippingLocationCell.swift

```diff
@@ -93,32 +99,23 @@ final class PledgeShippingLocationCell: UITableViewCell, ValueCell {
   override func bindViewModel() {
     super.bindViewModel()
 
-    self.countryButton.rac.title = self.viewModel.location
-    self.amountLabel.rac.attributedText = self.viewModel.amount
-
-    self.viewModel.shippingIsLoading
-      .observeForUI()
-      .observeValues { [weak self] isLoading in
-        self?.animate(isLoading)
-      }
-
-    self.viewModel.outputs.selectedShippingRule
-      .observeForUI()
-      .observeValues { [weak self] rule in
-        guard let self = self else { return }
-        self.delegate?.pledgeShippingCell(self, didSelectShippingRule: rule)
-      }
+    self.amountLabel.rac.attributedText = self.viewModel.outputs.amountAttributedText
+    self.shippingLocationButton.rac.title = self.viewModel.outputs.shippingLocation
   }
 
   // MARK: - Configuration
 
-  func configureWith(value: (project: Project, reward: Reward)) {
-    self.viewModel.inputs.configureWith(project: value.project, reward: value.reward)
+  func configureWith(value: (isLoading: Bool, project: Project, selectedShippingRule: ShippingRule?)) {
+    self.viewModel.inputs.configureWith(
+      isLoading: value.isLoading,
+      project: value.project,
+      selectedShippingRule: value.selectedShippingRule
+    )
   }
 
-  // MARK: - Public Functions
+  // MARK: - Actions
 
-  func animate(_: Bool) {}
```

Comment:
We don't need this for now...there's a card on our sprint board to address the loading state .. in the future this should be driven from the VM not externally